network-api/networkapi/wagtailpages/templates/buyersguide/bg_base.html

```diff
@@ -94,9 +94,11 @@
 
           {% for cat in categories %}
             {% with original=cat.original %}
-            {% if original.published_product_page_count > 0 %}
-              {% localizedroutablepageurl home_page 'category-view' original.slug as cat_url %}
-              <a class="multipage-link {% check_active_category current_category cat %}{% if original.featured is True %} featured{% endif %}" href="{{ cat_url }}" data-name="{{ cat.name }}">{{ cat.name }}</a>
+            {% if cat.parent == None %}
+              {% if original.published_product_page_count > 0 %}
+                {% localizedroutablepageurl home_page 'category-view' original.slug as cat_url %}
+                <a class="multipage-link {% check_active_category current_category cat %}{% if original.featured is True %} featured{% endif %}" href="{{ cat_url }}" data-name="{{ cat.name }}">{{ cat.name }}</a>
```

Comment:
we'll want to extend this tag so that it returns `active` for "this category matches, _or_ this category's `.parent` matches". That way, the nav will be correctly preselected already even if the JS doesn't kick in immediately

Reply:
```python
# Determine if a category nav link should be marked active
@register.simple_tag(name='check_active_category')
def check_active_category(current_category, target_category):
    # because we're working with potentially localized data,
    # make sure to compare the linguistic originals.
    current_category = getattr(current_category, 'original', current_category)
    target_category = getattr(target_category, 'original', target_category)
    match = current_category == target_category or current_category.parent == target_category
    return 'active' if match else ''
```